protobuf/lib/src/protobuf/pb_list.dart

```diff
@@ -11,7 +11,21 @@ typedef CheckFunc<E> = void Function(E? x);
 
 /// A [ListBase] implementation used for protobuf `repeated` fields.
 class PbList<E> extends ListBase<E> {
+  /// The actual list storing the elements.
+  ///
+  /// Note: We want only one [List] implementation class to be stored here to
+  /// make sure the list operations are monomorphic and can be inlined. In
+  /// constructors make sure initializers for this field all return the same
+  /// implementation class. (e.g. `_GrowableList` on the VM)
   final List<E> _wrappedList;
+
+  /// A growable list, to be used in `unmodifiable` constructor to avoid
+  /// allocating a list every time.
+  ///
+  /// We can't use `const <E>[]` as it makes the `_wrappedList` field
```

Comment:
We cannot have `const <E>[]`. Constant expressions in dart can only refer to instantiated types, not free type parameters.

-> Should be `const <Never>[]`

Reply:
updated the comment with `const []`.